pkg/terraform/terraform.go

```diff
@@ -309,20 +398,58 @@ func (c *Commander) BulkPlan() error {
 			c.cmdEnv = append(os.Environ(), "KUBE_CTX=live-1.cloud-platform.service.justice.gov.uk")
 			c.Workspace = "live-1"
 
-			err := c.Plan()
+			log.WithFields(log.Fields{"dir": dir}).Info("Setting kubectl current-context with: kubectl config use-context live-1.cloud-platform.service.justice.gov.uk")
+
+			err := c.kubectlUseContext("live-1.cloud-platform.service.justice.gov.uk")
+			if err != nil {
+				return err
+			}
+			cluster, err := c.kubectlCurContext()
+			if err != nil {
+				return err
+			}
+
+			if cluster == c.Workspace {
```

Comment:
I worry about this conditional statement as my cluster variable would output to `arn:aws:.....:cluster/manager` and my workspace will output to `manager`.

Reply:
Maybe this has to be noted upstream, the context name MUST equal xyz.

Reply:
> I worry about this conditional statement as my cluster variable would output to `arn:aws:.....:cluster/manager` and my workspace will output to `manager`.

@jasonBirchall , the kubeconfig we have in the s3 bucket has the context stored in format manager.cloud-platform.service... so the output of `kubectlCurContext` would return manager and the workspace return manager. I will add a note saying this might break if the kubeconfig format for context is not in the this format.